src/Http/Controllers/VoyagerDatabaseController.php

```diff
@@ -309,6 +309,36 @@ public function updateBread(Request $request, $id)
         }
     }
 
+    public function addBreadRelationship(Request $request, $id)
+    {
+        Voyager::canOrFail('browse_database');
+
+        $validator = Validator::make($request->all(), [
+            'details' => [
+                'required',
+                'json',
+                'regex:/^.*(?=.*\brelationship\b)(?=.*\bkey\b)(?=.*\blabel\b)(?=.*\bmethod\b).*$/'
+            ],
+            'type' => 'required',
+        ]);
+
+        if ($validator->fails()) {
+            return back()->with($this->alertException($validator->errors(), 'Update Failed'));
+        }
+
```

Comment:
Just realized , this will fail. 
`alertException()` expects an `Exception`, `$validator->errors()` returns a `MessageBag`